tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Qiskit Gradient Framework using Primitives\n",
+    "This tutorial demonstrates the use of the `qiskit.algorithms.gradients` module to evaluate quantum gradients using the [Qiskit Primitives](https://qiskit.org/documentation/apidoc/primitives.html).\n",
+    "\n",
+    "## Introduction\n",
+    "The gradient frameworks allows the evaluation of quantum gradients.\n",
+    "Besides gradients of expectation values of the form\n",
+    "$$ \\langle\\psi\\left(\\theta\\right)|\\hat{O}\\left(\\omega\\right)|\\psi\\left(\\theta\\right)\\rangle $$\n",
+    "the gradient frameworks also support the evaluation of the Quantum Geometric Tensor (QGT) and Quantum Fisher Information (QFI) of quantum states $|\\psi\\left(\\theta\\right)\\rangle$."
```

Comment:
The gradient framework allows to compute gradients of expectation values (mentioned), sampling probabilities (not mentioned), as well as the evaluation of metrics such as the QGT and QFI. So we should add the sampler gradients to this introduction:

```suggestion
    "The gradient frameworks allows the evaluation of quantum gradients.\n",
    "Besides gradients of expectation values of the form\n",
    "$$ \\langle\\psi\\left(\\theta\\right)|\\hat{O}\\left(\\omega\\right)|\\psi\\left(\\theta\\right)\\rangle $$\n",
    "and sampling probabilities of the form\n",
    "$$p_j(\\theta) = |\\langle j | \\psi(\\theta)\\rangle|^2$$\n"
    "the gradient frameworks also support the evaluation of the Quantum Geometric Tensor (QGT) and Quantum Fisher Information (QFI) of quantum states $|\\psi\\left(\\theta\\right)\\rangle$."
```

I have added the suggestion directly into the raw notebook, in theory you should be able to directly apply it by clicking on "commit suggestion" below, if you want to. It's also a good idea to format it locally to make sure it looks good.